lib/model/query/entities.js

```diff
@@ -744,16 +744,12 @@ resolveConflict.audit = (entity, dataset) => (log) => log('entity.update.resolve
 ////////////////////////////////////////////////////////////////////////////////
 // SERVING ENTITIES
 
-const _exportUnjoiner = unjoiner(Entity, Entity.Def, Entity.Extended.into('stats'), Actor.alias('actors', 'creator'));
```

Comment:
Can we remove the `Entity.Extended` frame altogether in lib/model/frames/entity.js? I think we should at least modify it to remove its `updates` field.

Reply:
That seems to be the only place `Entity.Extended` was used, so I will remove it.